backup_and_restore/application_backup_and_restore/features-and-plugins/oadp-configuring-velero-plugins.adoc

```diff
@@ -0,0 +1,75 @@
+:_content-type: CONCEPT
```

Comment:
```suggestion
:_content-type: ASSEMBLY
```
Per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata, assemblies should always use the assembly concept type